packages/cli/commands/functions/deploy.js

```diff
@@ -62,6 +62,13 @@ const loadAndValidateOptions = async options => {
 };
 
 const logBuildOutput = async resp => {
+  const { cdnUrl } = resp;
+
+  if (!cdnUrl) {
+    logger.debug('Unable to display build output. No cdnUrl was provided.');
```

Comment:
Nit: `cdnUrl` seems like an implementation detail that we don't need to pass along

Reply:
good point, I will make this message more generic